src/components/FilmCard/FilmCard.tsx

```diff
@@ -4,15 +4,15 @@ import { FilmPreview } from '../FilmPreview';
 
 import css from './FilmCard.scss';
 
-export function FilmCard({ preview, name, genre, year }: IFilm) {
+export function FilmCard({ poster_path, title, genres, release_date }: IFilm) {
```

Comment:
может, лучше принять одну проперти с фильмом?

```suggestion
export function FilmCard({ film }: { film: IFilm }) {
```

Reply:
Думаю, так нагляднее, какие проперти нам будут нужны. + не пишем везде film.